gcmt/tests/test_manipulations.py

```diff
@@ -1,7 +1,3 @@
-import numpy as np
-import xarray
-
-from gcmt.tests.test_gcmtools_common import *
```

Comment:
We need `all_raw_testdata`. Might have been deleted by the IDE when running optimize imports...
```from .test_gcmtools_common import all_raw_testdata```